src/journal/journald-server.c

```diff
@@ -1275,6 +1276,9 @@ int server_flush_to_var(Server *s, bool require_flag_file) {
         if (!s->system_journal)
                 return 0;
 
+        /* Reset current seqnum data to avoid unnecessary rotation when switching to system journal */
+        zero(*s->seqnum);
```

Comment:
i don't really follow here.

the idea was that during one boot process we stick to the same series of seqnums. now you are resetting the whole thing? why?